README.md

```diff
@@ -33,27 +15,53 @@ Get prices rates:
 Open new terminal window in application source and launch "curl" request with parameters:
 
 ```
-curl "http://127.0.0.1:5000/rates?date_from=2016-01-01&date_to=2016-01-10&origin=CNSGH&destination=NLRTM"
+docker exec -it rates_api curl "http://127.0.0.1:8000/rates?date_from=2016-01-01&date_to=2016-01-10&origin=CNSGH&destination=NLRTM"
+```
+or
+```
+docker exec -it rates_api curl "http://localhost:8000/rates?date_from=2016-01-01&date_to=2016-01-10&origin=CNSGH&destination=NLRTM"
```

Comment:
would it make sense to include `curl` calls without using container shell? imo pretty much everyone has `curl` locally